misc/images/ubuntu-base/Dockerfile

```diff
@@ -11,6 +11,10 @@ FROM ubuntu:noble-20250127
 
 # Ensure any Rust binaries that crash print a backtrace.
 ENV RUST_BACKTRACE=1
+# Disable backtrace collection in Rust libraries, such as `anyhow`. Collecting
+# backtraces at runtime can be expensive and is known to cause deadlocks
+# (database-issues#9159).
+ENV RUST_LIB_BACKTRACE=0
```

Comment:
Should we set it for things like `bin/environmentd`, too? Might be good for consistency, although not strictly required.

Reply:
I think we don't need to! `bin/environmentd` doesn't set `RUST_BACKTRACE=1`, so we shouldn't be collecting backtraces in `anyhow` in the first place.